PluginManager/po/it-local.po

```diff
@@ -182,17 +183,16 @@ msgid "Show hidden items"
 msgstr "Mostra gli elementi nascosti"
 
 #: PluginManager/PluginManager.py:502
-#, fuzzy
 msgid "Show Built-in items"
 msgstr "Mostra gli elementi interni"
 
 #: PluginManager/PluginManager.py:509
 msgid "* indicates 3rd party addon"
-msgstr "* indica gli addons di terze parti"
+msgstr "Gli elementi '*' sono forniti da autori di terze parti"
```

Comment:
The change requested here for about addons, not the *:
'* indica i componenti aggiuntivi di terze parti"